lms/static/scripts/frontend_apps/utils/google-picker-client.js

```diff
@@ -2,7 +2,16 @@ import { loadLibraries } from './google-api-client';
 
 export const GOOGLE_DRIVE_SCOPE = 'https://www.googleapis.com/auth/drive';
 
-/** @param {string} origin */
+/**
+ * Convert a domain (example.com) into a valid web origin (https://example.com).
```

Comment:
Very helpful, especially given that the function is called `addHttps` and it isn't immediately clear that what it's doing is generating an origin!